ui/src/main/java/edu/wpi/grip/ui/components/ExceptionWitnessResponderButton.java

```diff
@@ -127,22 +126,33 @@ private void assignFromExceptionEvent(ExceptionEvent event) {
      */
     @Inject
     ExceptionWitnessResponderButton(@Assisted Object origin, @Assisted String popOverTitle) {
-        super(null, addFadeTransition(
-                new Glyph("FontAwesome", FontAwesome.Glyph.EXCLAMATION_TRIANGLE)
-                        .color(Color.RED).size(DPIUtility.MINI_ICON_SIZE)));
+        super();
+
         this.origin = checkNotNull(origin, "The origin can not be null");
         this.popOverTitle = checkNotNull(popOverTitle, "The pop over title can not be null");
         this.tooltip = new Tooltip();
         this.getStyleClass().add(STYLE_CLASS);
 
-        this.setOnAction(event -> getPopover().show(this));
+        ImageView icon = new ImageView("/edu/wpi/grip/ui/icons/warning.png");
+
+        setOnMouseClicked(event -> getPopover().show(this));
+        setVisible(false);
+        setContentDisplay(ContentDisplay.GRAPHIC_ONLY);
+        setGraphic(icon);
+        icon.setFitWidth(DPIUtility.SMALL_ICON_SIZE);
+        icon.setFitHeight(DPIUtility.SMALL_ICON_SIZE);
 
-        this.setVisible(false);
+        FadeTransition ft = new FadeTransition(Duration.millis(500), icon);
```

Comment:
Why did you reduce this in time? It was 750 originally. It made it look smoother.
To be honest why did you modify the fade transition code at all? It looked fine the way it was.